tests/integration/relations/pgbouncer_provider/test_pgbouncer_provider.py

```diff
@@ -315,6 +319,25 @@ async def test_read_only_endpoint_in_scaled_up_cluster(ops_test: OpsTest):
         )
 
 
+@pytest.mark.client_relation
+async def test_legacy_relation_compatibility(ops_test: OpsTest):
```

Comment:
Does this test exist to check that both legacy and modern relations can be established in the same pgbouncer application?

Reply:
Yep. Happy to delete this test if this functionality is unnecessary, but otherwise I figured it was worth testing :)

Reply:
I think it's good to have it. Should you update the phrase `Using legacy relations with normal relations is currently not tested.` on README as you're testing it?